CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
   - JDBCRecipientRewriteTable implementation. Please use JPARecipientRewriteTable instead.
   - JamesUsersJdbcRepository, DefaultUsersJdbcRepository. Please use JpaUsersRepository instead.
   - MailboxQuotaFixed matcher. Please use IsOverQuota instead.
+- UsersFileRepository, which was marked as deprecated for years
```

Comment:
indent issue?

Reply:
No actually this removal was not planned as part of 3.4.0 hence the level of indent is right.